### DISCUSSION TITLE
Restricting SSL Ciphers without using nginx as reverse proxy

### DISCUSSION THREAD

**Discussion by Plaschke-Culturall** [requarks/wiki#7220] [Help / Questions] [answered]
When we test our wiki.js installation on www.ssllabs.com/ssltest/ we can see that a couple of weak cipher suites are accepted by our server.

We don't have nginx installed and currently we use letsencrypt (not custom) as ssl provider:
```
  # Provider to use, possible values: custom, letsencrypt
  provider: letsencrypt
```

Is it possible to restrict the allowed ciphers somewhere in the config? 
Or do we have to install nginx for that?

**Comment by NGPixel** [comment, marked as answer]
Use nginx if you need a custom configuration. The ciphers are those enabled by default in Node.js.
